Formula/lib/libbi.rb

```diff
@@ -128,6 +128,12 @@ def install
 
     resource("thrust").stage { (include/"thrust").install Dir["thrust/*"] }
 
+    # make Homebrew packages discoverable for compilation/linking
+    inreplace "share/configure.ac" do |s|
+      s.gsub!(/^(CPPFLAGS=.*)"$/, "\\1 -I#{HOMEBREW_PREFIX}/include\"")
+      s.gsub!(/^(LDFLAGS=.*)"/, "\\1 -I#{HOMEBREW_PREFIX}/lib\"")
```

Comment:
Let's use `s.read_make_var` and `s.change_make_var!` here. Also, the `LDFLAGS` flag should be `-L`.

Reply:
Thanks, done.